src/Data/Textile/Simulator.elm

```diff
@@ -210,6 +214,37 @@ computeDurability ({ inputs } as simulator) =
     }
 
 
+computeMakingAirTransportRatio : Simulator -> Simulator
+computeMakingAirTransportRatio ({ inputs } as simulator) =
+    simulator
+        |> updateLifeCycleStep Label.Making
+            (\({ country } as step) ->
+                { step
+                    | airTransportRatio =
+                        let
+                            iSEuropeOrTurquey =
+                                country.zone == Zone.Europe || country.code == Country.codeFromString "TR"
+                        in
+                        case ( inputs.airTransportRatio, iSEuropeOrTurquey, Unit.floatDurabilityFromHolistic simulator.durability ) of
+                            -- Value provided by the user should always precede
+                            ( Just _, _, _ ) ->
+                                step.airTransportRatio
+
+                            -- If Making country is Europe or Turquey, airTransportRatio should always be 0
+                            ( _, True, _ ) ->
+                                Split.zero
+
+                            -- Else it should depend on the computed durability
+                            ( _, _, durabilty ) ->
+                                if durabilty >= 1 then
+                                    Split.fromFloat 0.33 |> Result.withDefault step.airTransportRatio
+
+                                else
+                                    Split.full
```

Comment:
Nit: Probably matter of taste but I'd find this block more readable using plain old if statements:

```suggestion
                        if inputs.airTransportRatio /= Nothing then
                            -- User-provided value always takes precedence
                            step.airTransportRatio

                        else if country.zone == Zone.Europe || country.code == Country.codeFromString "TR" then
                            -- If Making country is Europe or Turquey, airTransportRatio is always 0
                            Split.zero

                        else if Unit.floatDurabilityFromHolistic simulator.durability >= 1 then
                            -- Durable garments outside of Europe and Turquey
                            Split.fromFloat 0.33
                                |> Result.withDefault step.airTransportRatio

                        else
                            Split.full
```